src/NServiceBus.Core.Tests/Faults/MoveFaultsToErrorQueueTests.cs

```diff
@@ -159,18 +164,23 @@ MoveFaultsToErrorQueueBehavior CreateBehavior(TransportTransactionMode transacti
         {
             var behavior = new MoveFaultsToErrorQueueBehavior(
                 criticalError,
-                errorQueueAddress,
-                "public-receive-address",
                 transactionMode,
                 new FailureInfoStorage(10));
 
             return behavior;
         }
 
-        static Task CaptureFaultContext(IFaultContext ctx, out IFaultContext context)
+        static Task CaptureFaultContext(IFaultContext ctx, out IFaultContext context, string errorQueueAddress, string localAddress)
         {
             context = ctx;
-            return TaskEx.CompletedTask;
+
+            var chain = new BehaviorChain(new[]
```

Comment:
This is a hack to fix the existing tests, but perhaps these tests shouldn't be here now...